src/stub/core_impl/base.rs

```diff
@@ -332,15 +349,59 @@ impl<T: Target, C: Connection> GdbStubImpl<T, C> {
                 }
                 HandlerStatus::NeedsOk
             }
+            Base::qC(_) => {
+                res.write_str("QC")?;
+                let pid = self.get_sane_current_pid(target)?;
+
+                match target.base_ops() {
+                    BaseOps::SingleThread(_) => res.write_specific_thread_id(SpecificThreadId {
+                        pid: self
+                            .features
+                            .multiprocess()
+                            .then_some(SpecificIdKind::WithId(pid)),
+                        tid: SpecificIdKind::WithId(SINGLE_THREAD_TID),
+                    })?,
+                    BaseOps::MultiThread(ops) => {
```

Comment:
What should it do if it is set to `SINGLE_THREAD_TID`? I've set it so that it does the previous behaviour of looking through all threads, but I can change it to something else if you prefer...

Reply:
Nope, that's exactly the behavior I would've expected - you're good.